modules/gdscript/gdscript.cpp

```diff
@@ -2368,62 +2368,60 @@ void GDScriptLanguage::frame() {
 
 /* EDITOR FUNCTIONS */
 void GDScriptLanguage::get_reserved_words(List<String> *p_words) const {
-	// TODO: Add annotations here?
 	static const char *_reserved_words[] = {
-		// operators
+		// Control flow.
+		"if",
+		"elif",
+		"else",
+		"match",
+		"for",
+		"when",
+		"while",
+		"pass",
+		"break",
+		"continue",
+		"return",
+		// Declarations.
+		"static",
+		"var",
+		"const",
+		"enum",
+		"func",
+		"signal",
+		"class",
+		"class_name",
+		"extends",
+		// Other keywords.
+		"await",
+		"breakpoint",
+		"self",
+		"super",
+		// Operators.
 		"and",
-		"in",
-		"not",
 		"or",
-		// types and values
-		"false",
-		"float",
-		"int",
-		"bool",
+		"not",
+		"in",
+		"as",
+		"is",
+		// Constants.
 		"null",
+		"false",
+		"true",
 		"PI",
 		"TAU",
 		"INF",
 		"NAN",
-		"self",
-		"true",
+		// Types.
 		"void",
-		// functions
-		"as",
+		// Functions.
 		"assert",
-		"await",
-		"breakpoint",
-		"class",
-		"class_name",
-		"extends",
-		"is",
-		"func",
 		"preload",
-		"signal",
-		"super",
-		// var
-		"const",
-		"enum",
-		"static",
-		"var",
-		// control flow
-		"break",
-		"continue",
-		"if",
-		"elif",
-		"else",
-		"for",
-		"pass",
-		"return",
-		"match",
-		"while",
-		"when",
 		// These keywords are not implemented currently, but reserved for (potential) future use.
 		// We highlight them as keywords to make errors easier to understand.
 		"trait",
 		"namespace",
 		"yield",
-		nullptr
+		nullptr,
```

Comment:
keep the standard in the project where the last item does not have a trailing comma.

Reply:
As far as I know, there is no such standard. You can check this with a regular expression `,\n\t+}`. You'll find many arrays and enums with a trailing comma, even if no new elements are expected to be added after the last element. For example `MAX,\n\t+}` found 107 matches.

Reply:
I'd say if there is an enforceable rule it would be to add comma always. This is the most VCS-friendly style.